box64/box64.spec

```diff
@@ -0,0 +1,761 @@
+# Tests are disabled as they require x86_64 libraries to run
+%bcond tests 0
+
+%global forgeurl https://github.com/ptitSeb/box64
+
+%global common_description %{expand:
+Box64 lets you run x86_64 Linux programs (such as games) on non-x86_64 Linux
+systems, like ARM (host system needs to be 64-bit little-endian).}
+
+%define hash a9f4e72c4b4f15289c8eef0b9be0359e2acf50dc
+
+Name:           box64
+Version:        %{hash}
```

Comment:
I believe the version also needs to be specified.

For example, if this hash starts with `2` but the next version starts with `1`, it will be seen as a downgrade.

Perhaps we can include the version (Box64 uses odd numbers for unreleased versions and even for pinned releases). Right now this would look like:

```
Version: 0.37.0-%{hash}
```

Reply:
Actually, my suggestion could still run into the same problem. Perhaps we also want the date?

```
Version: 0.37.0-20250819-%{hash}
```